Barotrauma/BarotraumaShared/SharedSource/Characters/AI/Objectives/AIObjectiveIdle.cs

```diff
@@ -382,12 +382,18 @@ public void Wander(float deltaTime)
                     {
                         foreach (Item item in Item.ItemList)
                         {
-                            if (item.CurrentHull != currentHull || !item.HasTag(Tags.ChairItem)) { continue; }
-                            //not possible in vanilla game, but a mod might have holdable/attachable chairs
+                            
+                            if (item.CurrentHull != currentHull) { continue; }
```

Comment:
The removal of the Chair tag check is a problem for performance. Previously this line allowed the loop to skip non-chair items early, now it continues much further into the method and does the somewhat expensive `GetComponent` call (expensive for such a tight loop at least). We have done an optimization to this method in our private repo that makes this loop only search through items with the chair tag, since enumerating through the entire ItemList can get very expensive. That change conflicts with this one, so I think we'll need to wait until it's merged before working on this further.